### PR TITLE
Fix: fetch kyma info

### PR DESCRIPTION
**Pull request by petersutter**
**What this PR does / why we need it**:
Currently the kyma info is not properly loaded. With this PR the kyma info is fetched when the details page is shown / on reload.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement user

```

### CODE REVIEW

File: frontend/src/pages/ShootDetails.vue [hunk 1]
```diff
@@ -146,20 +146,24 @@ export default {
   methods: {
     ...mapActions([
       'getShootAddonKyma'
-    ])
+    ]),
+    fetchKymaInfo () {
+      if (!this.isKymaFeatureEnabled && this.isKymaAddonEnabled) {
```
Comment (holgerkoser) [inline frontend/src/pages/ShootDetails.vue]: KymaAddon is disabled but we fetch the KymaInfo?

File: frontend/src/pages/ShootDetails.vue [hunk 2]
```diff
@@ -146,20 +146,24 @@ export default {
...
       this.$refs.shootLifecycle.showHibernationConfigurationDialog()
     }
+    this.fetchKymaInfo()
   },
   watch: {
     isKymaAddonEnabled (currentlyEnabled, previouslyEnabled) {
-      if (!this.isKymaFeatureEnabled) {
-        return
-      }
       if (!previouslyEnabled && currentlyEnabled) {
```
Comment (holgerkoser) [inline frontend/src/pages/ShootDetails.vue]: why check the previous value? If current value is true and it has changed we should fetch the infi.

File: frontend/src/pages/ShootDetails.vue (hunk 1 above)
Comment (petersutter) [reply frontend/src/pages/ShootDetails.vue]: right, will fix it

File: frontend/src/pages/ShootDetails.vue (hunk 2 above)
Comment (petersutter) [reply frontend/src/pages/ShootDetails.vue]: ✅

**Pull request merged**